assets/scss/pages/_author.scss

```diff
@@ -21,6 +21,10 @@
         }
       }
     }
+
+    .share-buttons {
+      float: none;
```

Comment:
That affects L, XL screens as well. We need the share buttons to float right there. See `_share-buttons.scss`.